internal/infrastructure/price-feeder/repository.go

```diff
@@ -0,0 +1,25 @@
+package pricefeederinfra
+
+import (
+	"context"
+)
+
+type PriceFeedRepository interface {
```

Comment:
you're totally right in defining this as repository but I'm afraid it could create confusion with domain repos.

What do you think of calling this?
```suggestion
type PriceFeedStore interface {
```

Reply:
Done